src/objects/Application__c.object

```diff
@@ -0,0 +1,268 @@
+            </valueSetDefinition>
+        </valueSet>
+    </fields>
+    <fields>
+        <fullName>Application_Type__c</fullName>
+        <description>The type of program or track the applicant is applying to. For example, Undergraduate, Graduate, or Transfer.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The type of program or track the applicant is applying to. For example, Undergraduate, Graduate, or Transfer.</inlineHelpText>
+        <label>Application Type</label>
+        <required>false</required>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Picklist</type>
+        <valueSet>
+            <restricted>true</restricted>
+            <valueSetDefinition>
+                <sorted>false</sorted>
+                <value>
+                    <fullName>Undergrad</fullName>
+                    <default>false</default>
+                    <label>Undergrad</label>
+                </value>
+                <value>
+                    <fullName>Transfer</fullName>
+                    <default>false</default>
+                    <label>Transfer</label>
+                </value>
+                <value>
+                    <fullName>Graduate</fullName>
+                    <default>false</default>
+                    <label>Graduate</label>
+                </value>
+                <value>
+                    <fullName>Post-Doc</fullName>
+                    <default>false</default>
+                    <label>Post-Doc</label>
+                </value>
+            </valueSetDefinition>
+        </valueSet>
+    </fields>
+    <fields>
+        <fullName>Applying_To__c</fullName>
+        <deleteConstraint>SetNull</deleteConstraint>
+        <description>The academic program or major the applicant is applying to.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The academic program or major the applicant is applying to.</inlineHelpText>
+        <label>Applying To</label>
+        <referenceTo>Account</referenceTo>
+        <relationshipLabel>Applications</relationshipLabel>
+        <relationshipName>Applications</relationshipName>
+        <required>false</required>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Lookup</type>
+    </fields>
+    <fields>
+        <fullName>Preparer__c</fullName>
+        <deleteConstraint>SetNull</deleteConstraint>
+        <description>The Contact preparing the application, often a guardian.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The Contact preparing the application. For example, a guardian or a person assisting someone with special needs.</inlineHelpText>
+        <label>Preparer</label>
+        <referenceTo>Contact</referenceTo>
+        <relationshipLabel>Applications (Preparer)</relationshipLabel>
+        <relationshipName>Applications1</relationshipName>
```

Comment:
May want to consider this API relationship name being something like Applications_Preparer? (Thoughts @lifewithryan ?)

Reply:
Yes, let's move with Bobby's suggestion. I thought at one point the spec called that out, but I'm betting in the numerous changes, copy/pastes to different versions it got dropped somewhere along the way.

Reply:
I think this is only used in code when referring a relationship in query, but I guess it does not hurt to give it a better api name.

Reply:
Although, I assume we need to change the applicant relationship name to Applications_Applicant too?

Reply:
Also...should we better call it Application_Preparers? @lifewithryan @robertbwatson

Reply:
I think Applications Preparer is actually more correct. When viewing a contact record, it will be the related list of Applications that this contact was the Preparer for. So multiple applications but one preparer (which is the contact we are viewing). Same for the Applicant version.